pkg/api/customization/kontainerdriver/formatter.go

```diff
@@ -26,15 +26,23 @@ func NewFormatter(manangement *config.ScaledContext) types.Formatter {
 	return format.Formatter
 }
 
-const clusterByKontainerDriverKey = "clusterbyKontainerDriver"
+const clusterByGenericEngineConfigKey = "genericEngineConfig"
 
-func clusterByKontainerDriver(obj interface{}) ([]string, error) {
+// pass in a cluster object, if it has a genericEngineConfig return the driverName as a field (kd-*)
```

Comment:
Drop `(kd-*)` this is only valid for imported, we just use the ID/name for drivers